docs/se/webserver/11_access-log.adoc

```diff
@@ -166,4 +166,4 @@ io.helidon.webserver.accesslog.AccessLogHandler.append=true
 io.helidon.webserver.AccessLog.level=INFO
 io.helidon.webserver.AccessLog.useParentHandlers=false
 io.helidon.webserver.AccessLog.handlers=io.helidon.webserver.accesslog.AccessLogHandler
-----
+----
```

Comment:
This looks like a white space change. This should be displayed in the diff as "File renamed without changes"